src/ui/PageContentsForIndex/PageContentsForIndex.tsx

```diff
@@ -47,7 +47,7 @@ const exampleUrlByPackageAndDestination = {
   react: {
     "": "https://www.npmjs.com/package/react",
     h: "https://reactjs.org",
-    s: "https://github.com/facebook/react/tree/master/packages/react",
+    s: "https://github.com/facebook/react/tree/main/packages/react",
```

Comment:
Despite my other comment on master / main branches, the diff in this file makes sense 👍